todo_app/test_integration.py

```diff
@@ -0,0 +1,48 @@
+import os
+import requests
+from todo_app import app
+from unittest.mock import patch, Mock
+from dotenv import load_dotenv, find_dotenv
+
+import pytest
+from dotenv import load_dotenv, find_dotenv
+from todo_app.app import create_app
+
+# @pytest.fixture
+# def test_evironment_vars():
+#     file_path = find_dotenv('env.test')
+#     load_dotenv(file_path, override=True)
```

Comment:
Let's delete rather than commenting out

Reply:
I forgot to delete this piece of code :)